style.css

```diff
@@ -529,3 +529,212 @@ p {
 a {
   text-decoration: none;
 }
+
+.footer {
+  background: #e5e5e5;
+}
+
+.containers {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 90px;
+  position: absolute;
+  width: 360px;
+  height: 844px;
+  left: 0;
+  top: 6038px;
+  background: #e5e5e5;
+}
+
+.textheads {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 320px;
+  height: 216px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.contactheads {
+  width: 320px;
+  height: 220px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.myforms {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] Kindly make sure the form is at the bottom of the page as it is in the template in Figma.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193681016-228098cd-772c-4006-8eea-9c91dd29df7a.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193681118-3be3e04e-7dc9-4518-bfeb-cb898516d7ae.png)